package.json

```diff
@@ -8,8 +8,10 @@
   },
   "license": "MIT",
   "dependencies": {
+    "crypto": "^1.0.1",
```

Comment:
We use that only in the tests correct? Then it is a dev-dependency

Reply:
implemented